swap_meet/vendor.py

```diff
@@ -1,2 +1,91 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        self.inventory = inventory if inventory else []
+
+    def add(self,item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
+    
+    def get_by_id(self, search_id):
+        if isinstance(search_id, int):
```

Comment:
Great to see you thinking about data validation in a more practical setting. We could actually move this logic into a separate function. Though functionally sound here, by performing input validation inside this function we start to muddy the concern of the `get_by_id` function. Which is to just return an item by `id`, not validate the `id`. Separating the two concerns makes the functions more readable as well as being easier to maintain. Try searching the term *Separation of Concerns*.